pages/linux/iostat.md

```diff
@@ -1,19 +1,27 @@
 # iostat
 
-> Report stats for devices and partitions.
+> Report statistics for devices and partitions.
 
-- Display disk statistics with disk IDs in human readable format:
+- Display a report of CPU and disk statistics since system startup:
 
-`iostat -h`
+`iostat`
 
-- Display disk statistics with disk names (including LVM) in human readable format:
+- Display a report of CPU and disk statistics since system startup with units converted to megabytes:
```

Comment:
Considering that this example comes right after the basic one with no flags, perhaps the description could dispense with repeating the "since system startup"? That would make it shorter and easier to read, IMO.